docs/application/dotnet/guides/connectivity/wifi.md

```diff
@@ -7,7 +7,11 @@ The main features of the Tizen.Network.WiFi namespace include:
 
 -   Wi-Fi device management
 
-    You can use the [Tizen.Network.WiFi.WiFiManager](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Network.WiFi.WiFiManager.html) class to control aspects of your application's wireless connection. For example, you can [activate](#activating_wifi_device) or [deactivate](#deactivating_wifi_device) Wi-Fi, [monitor connection state changes](#managing_events), and [scan for available access points](#accesspoint_scan).
+    You can use the [Tizen.Network.WiFi.WiFiManager](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Network.WiFi.WiFiManager.html) class to control aspects of your application's wireless connection.
+
```

Comment:
Suggestion:
Can you please verify line number 42 and 50. 
In line 42, the note tag style is not proper. It is recommended to use:
>**Note**
>
>You can test the Wi-Fi functionality on a target device only. The emulator does not support this feature.

In line 50, the `tizen-manifest.xml` is in code tag. To emphasize file names, it is recommended to use bold style. 
Please verify and update.

Reply:
Hello, @Reni-90 
In case of "Prerequisites" section, usually "tizen-manifest.xml" is highlighted with code style. (Using \`.) I think you can refer other pages to check it.

However, if you choosed bold font for the "file name", I think you should update every "Prerequisites" sections in all files. Also for that case, please use only one style. For now both bold style and code style are applied, like <strong>`tizen-manifest.xml`</strong>. I think this emphasizing is too much..

By the way, for the bold style, please guide to use \<strong\>, not \<b\>.

Reply:
@mijin85cho 
If it is better to keep `tizen-manifest.xml` in code tag. Then, we will continue. To change every document back will be difficult. 
Therefore, we will keep  `tizen-manifest.xml` in code tag only.